bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,187 @@
+# ferroamp Binding
+
+The ferroamp binding is used to get live data from ferroamp energy systems.
+
+The binding retrieve data from the different parts of the ferroamp energy system such as:
+
+- `Ehub`: EnergyHub Wall and EnergyHub XL.
+- `Sso`: Solar string optimizer.
+- `Eso`: Bidirectional DC/DC converter for connection of battery.
+- `Esm`: Energy Storage Module.
+ 
+
+## Supported Things
+
+This binding is compatible with EnergyHub Wall and EnergyHub XL, and use connection to your local Ehub via LAN or similar.
+Data and commands are sent using MQTT where the user connects to the MQTT broker residing on the ferroamp system.
+Contact ferroamp support to enable MQTT in the EnergyHub and to get Username and Password:
+https://ferroamp.com/om-ferroamp/
+
+## Discovery
+
+Discovery is not supported, as the system has a number of fixed parameters.
+
+## Thing Configuration
+
+Binding MQTT must be installed before binding ferroamp.
```

Comment:
This dependency is essentail and i think it might be usefull to move this line to the ## ferroamp binding section at the top.